lib/pf/Switch/Ruckus/SmartZone.pm

```diff
@@ -358,9 +358,15 @@ sub check_if_radius_request_psk_matches {
         return $FALSE;
     }
 
+    my $pmk = $self->cache->compute(
```

Comment:
Do you think it will be interesting to use the distributed cache ? (in case of a cluster with a large number of account)

Reply:
I deliberately picked redis because it was faster to access so if you have a lot of PMK to compare against, then the query time is much lower when we use Redis vs the DB